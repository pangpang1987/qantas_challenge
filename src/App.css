body, html, #root {
  height: 100%;
  color: #424242;
}

* {
  box-sizing: border-box;
}
