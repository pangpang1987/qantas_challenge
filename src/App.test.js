import React from "react";

import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import App from './App';

describe('Component App', () => {
  it('should render properly', () => {
    const component = shallow(<App />);

    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  })
})
