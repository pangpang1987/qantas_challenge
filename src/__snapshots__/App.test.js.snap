// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component App should render properly 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <BrowserRouter>
    <routes />
  </BrowserRouter>
</Provider>
`;
